Fix model assignment logic in Python code in 01_azureopenai.ipynb

m444x opened the pull request:
Updated the model assignment to correctly check if the environment variable 'AZURE_OPENAI_COMPLETION_MODEL' is None. The model will now default to 'gpt-4o-mini' only if the environment variable is not set.

m444x committed:
Fix model assignment logic in Python code in 01_azureopenai.ipynb

Updated the model assignment to correctly check if the environment variable 'AZURE_OPENAI_COMPLETION_MODEL' is None. The model will now default to 'gpt-4o-mini' only if the environment variable is not set.

sebafo commented:
Great find, thanks! A more elegant solution might be:
```
model = os.getenv("AZURE_OPENAI_COMPLETION_MODEL") or "gpt-4o-mini"
```
Would you like to update your commit?

m444x committed:
Refactor model assignment for efficiency

Changed the model assignment to use the 'or' operator. The model will now default to 'gpt-4o-mini' if the environment variable 'AZURE_OPENAI_COMPLETION_MODEL' is not set.

m444x commented:
Thank you, much better ;-)
